source/dofs/dof_tools_constraints.inst.in

```diff
@@ -29,6 +29,20 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension : SPACE_DIMENSIONS)
       const FullMatrix<double> &,
       const std::vector<unsigned int> &,
       const double);
+    template void DoFTools::make_periodicity_constraints(
+      const DoFHandler<deal_II_dimension,
+                       deal_II_space_dimension>::face_iterator &,
+      const DoFHandler<deal_II_dimension,
+                       deal_II_space_dimension>::face_iterator &,
+      AffineConstraints<float> &,
+      const ComponentMask &,
+      bool,
+      bool,
+      bool,
+      const FullMatrix<double> &,
+      const std::vector<unsigned int> &,
+      const float);
+
```

Comment:
I think this change in the number of instantiations and the one a few lines below should be in a separate PR. Would you mind opening that one so it can be reviewed separately?

Reply:
Yes, I agree!